lib/azure/armrest/hdinsight/hdinsight_application_service.rb

```diff
@@ -0,0 +1,11 @@
+module Azure
+  module Armrest
+    module HDInsight
+      class HDInsightApplicationService < ResourceGroupBasedSubservice
+        def initialize(armrest_configuration, options = {})
+          super(armrest_configuration, 'clusters', 'applications', 'Microsoft.HDInsight', options)
```

Comment:
`super` only accepts 4 arguments

Reply:
I think `ResourceGroupBasedSubService#initialize` takes five arguments, and that would mean a super call for that method would as well, yes? None of the arguments for its initalize method have defaults or anything

Reply:
My bad 👎 .  I did not notice this is a sub service.

Reply:
No worries, glad I didn't post something stupid for all the world to see 👍